### PR TITLE
Deprecate collection: Add changelog fragment

### PR DESCRIPTION
**Pull request by mariolenz**
##### SUMMARY
Follow-up to #624 / #626 in order to add a changelog entry.

##### ISSUE TYPE
- Docs Pull Request

##### ADDITIONAL INFORMATION
I guess it's easier to get this directly into the `stable-5` branch instead of into `main` and then backporting. But if you prefer this, just let me know and I'll create a new PR for `main`.

### CODE REVIEW

File: changelogs/fragments/626-deprecate-collection_changelog_fragment.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+deprecated_features:
+  - This collection and all content in it is unmaintained and deprecated
+    (https://forum.ansible.com/t/8030).
+    If you're interested in maintaining parts of the collection, please copy them to your own repository.
```
Comment (felixfontein) [inline changelogs/fragments/626-deprecate-collection_changelog_fragment.yml:4]:
```suggestion
    If you are interested in maintaining parts of the collection, please copy them to your own repository,
    and tell others about in the Forum discussion.
```

File: changelogs/fragments/626-deprecate-collection_changelog_fragment.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    See the collection creator path for details
+    https://docs.ansible.com/ansible/devel/dev_guide/developing_collections_path.html.
```
Comment (felixfontein) [inline changelogs/fragments/626-deprecate-collection_changelog_fragment.yml:6]:
```suggestion
    See the `collection creator path <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections_path.html>`__
    for details.
```

**Comment by mariolenz** [comment]
Thanks @felixfontein! I've committed your suggestions :smiley:

BTW do you know why `ansible/third-party-check` fails? `NODE_FAILURE` usually means there's some basic problem with Zuul, or do you sse anything I've done wrong in this PR to trigger this? This is not because I'm targeting `stable-5` directly, is it?

**Comment by felixfontein** [comment]
@mariolenz the Zuul problem is very likely unrelated to this PR. Let's simply ignore it.

**Comment by felixfontein** [comment]
I created a discussion on the forum for Zuul: https://forum.ansible.com/t/zuul-seems-to-be-down-right-now/10280

I hope I added the correct tags, now that we don't have that Matrix room anymore...

**Review by felixfontein — APPROVED** [review]

**Comment by mariolenz** [comment]
@Andersson007 What do you think about this? Merge it and then do a 5.1.0 release?

Although I won't do it. I'm just trying to help out, but since I haven't been involved with this collection in the past I don't want to do a release.

**Comment by mariolenz** [comment]
@felixfontein I think the `backport-5` label is wrong here because the PR is already targeting the `stable-5` branch.

**Comment by felixfontein** [comment]
@mariolenz oh, I didn't notice. In that case the PR should be changed so it's against `main`.

**Comment by felixfontein** [comment]
(Though I guess in the end it doesn't really matter, so :shrug:)

**Comment by Andersson007** [comment]
@mariolenz @felixfontein thanks!

Sure, I'll forthport (hehe) the PR to main and release it:)

**Pull request merged**